Archipelago_Gifting.cpp

```diff
@@ -15,23 +14,22 @@ extern Json::FastWriter writer;
 extern Json::Reader reader;
 extern std::mt19937_64 rando;
 extern int ap_player_team;
+extern int ap_player_id;
```

Comment:
`AP_GetPlayerID`? Or does that not work?

Reply:
we could but it bothered me that we would use a method call `AP_GetPlayerID` for the slot, but then access the `ap_player_team` directly? so in my head we could easily just use them both directly

Reply:
Philosophy here is to always use the user-available API when possible. That way, there's less chance of bugs when trying to access private data directly